integration_test/doc.go

```diff
@@ -1,18 +0,0 @@
-// Copyright 2016 Palantir Technologies, Inc.
```

Comment:
where did the integration tests go?

Reply:
No idea, I obviously wasn't paying attention to a `git add -A`.

Reply:
much better now! :)